.github/workflows/pull_request.generated.yml

```diff
@@ -15,13 +15,13 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
-    - name: Set Node.js 16.x
-      uses: actions/setup-node@v4
+    - uses: actions/checkout@v3
+    - name: Set Node.js 20.x
+      uses: actions/setup-node@v3
```

Comment:
why downgrade `actions/setup-node`?

Reply:
by accident. I regenerated wotkflows from templates, which don't have upgrades in them

Reply:
I'll just delete templates

Reply:
I didn't do that because that results in duplication and I don't want to derail this fix into converting to shared workflows